project/plugins.sbt

```diff
@@ -2,7 +2,7 @@ addSbtPlugin("org.scala-sbt" % "sbt-houserules" % "0.3.7")
 addSbtPlugin("org.scala-sbt" % "sbt-contraband" % "0.4.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.3")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "3.0.2")
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.12-rc5")
-libraryDependencies += "com.trueaccord.scalapb" %% "compilerplugin" % "0.6.0"
-addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.9")
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.15")
```

Comment:
Unfortunately we cannot do this, see https://github.com/sbt/zinc/issues/546

Reply:
ok reverted